projects/api/src/category/dto/update-category.dto.ts

```diff
@@ -0,0 +1,42 @@
+import { ApiProperty } from '@nestjs/swagger';
+import { IsNotEmpty, IsOptional, IsString, Matches, MaxLength, MinLength } from 'class-validator';
+
+class UpdateCategoryDto {
+  @ApiProperty({
+    type: 'string',
+    description: 'The name to update the category',
+    example: 'Mouse',
+  })
+  @IsString({
+    message: 'The category title must be a string',
+  })
+  @Matches(/^[A-Za-z\s]+$/, {
```

Comment:
Why you put more validations for the name here and not for the createDto, please put the same for both